cmd/geco-i2b2-data-source/data-source-plugin.go

```diff
@@ -0,0 +1,12 @@
+package main
+
+import (
+	"github.com/ldsec/geco-i2b2-data-source/pkg/datasource"
+	"github.com/ldsec/geco/pkg/sdk"
+)
+
+// DataSourceType is the type of the data source.
+var DataSourceType sdk.DataSourceType = "i2b2-medco"
```

Comment:
```suggestion
var DataSourceType sdk.DataSourceType = "i2b2"
```
we don't want to hear of medco here :stuck_out_tongue_closed_eyes:

Reply:
Actually that's a good question, do you really?

Because what is implemented in here is not only i2b2 stuff, but there is an additional layer of features (I'm thinking of cohorts e.g.). This is why I called that `i2b2-medco` and not just `i2b2` here. Given this, do you confirm you want to remove all references to medco or not?

Reply:
Right, then probably `i2b2-geco` is the right name at the moment